optin/puller/puller/db/CompletedOptinDatabase.py

```diff
@@ -45,6 +45,10 @@ def create_tables(self):
 			optin_id integer,
 			FOREIGN KEY (optin_id) REFERENCES optin_id(id)
 		)''')  # address cannot be unique because merges are supported
+		cursor.execute('''CREATE TABLE IF NOT EXISTS nem_block_timestamps (
```

Comment:
We need NEM timestamps also for InProgressOptinDatabase.optin_error and optin_request. did you plan to add it there too?